<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>operateDialog</class>
 <widget class="QDialog" name="operateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>操作</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(32, 30, 41);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="readControlBtn1Start">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(66, 79, 131);</string>
   </property>
   <property name="text">
    <string>开始读取</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readControlBtn2Start">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(66, 79, 131);</string>
   </property>
   <property name="text">
    <string>开始读取</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnBtn">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>390</y>
     <width>130</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(66, 79, 131);</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="shutdownBtn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>390</y>
     <width>130</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(66, 79, 131);</string>
   </property>
   <property name="text">
    <string>关机</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fanControlBtn_on">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>390</y>
     <width>130</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(66, 79, 131);</string>
   </property>
   <property name="text">
    <string>风扇开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fanControlBtn_off">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>390</y>
     <width>130</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(66, 79, 131);</string>
   </property>
   <property name="text">
    <string>风扇关</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readerRebootBtn1">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(66, 79, 131);</string>
   </property>
   <property name="text">
    <string>重启</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readerRebootBtn2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>80</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(66, 79, 131);</string>
   </property>
   <property name="text">
    <string>重启</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>116</width>
     <height>46</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>读写器1：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>116</width>
     <height>46</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>读写器2：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_info">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>165</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="readerSetBtn2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>80</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(66, 79, 131);</string>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readerSetBtn1">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(66, 79, 131);</string>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>241</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>允许的EPC字节数</string>
   </property>
   <widget class="QCheckBox" name="checkBox_2Byte">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>78</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>2字节</string>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_12Byte">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>87</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>12字节</string>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_16Byte">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>87</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>16字节</string>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_6Byte">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>78</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>6字节</string>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_8Byte">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>78</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>8字节</string>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4Byte">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>78</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>4字节</string>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="readControlBtn1Stop">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(66, 79, 131);</string>
   </property>
   <property name="text">
    <string>开始读取</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readControlBtn2Stop">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(66, 79, 131);</string>
   </property>
   <property name="text">
    <string>开始读取</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>returnBtn</sender>
   <signal>clicked()</signal>
   <receiver>operateDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
